src/Azure.Functions.Cli/StaticResources/Dockerfile.dotnet9Isolated

```diff
@@ -0,0 +1,16 @@
+FROM mcr.microsoft.com/dotnet/sdk:9.0-preview-noble-amd64 AS installer-env
```

Comment:
We should not be picking an architecture-specific image tag

Reply:
changed